src/types/translation.ts

```diff
@@ -1307,5 +1327,6 @@ export enum PlanSection {
   OPERATIONAL_SOLUTION_SUBTASKS = 'operationalSolutionSubtasks',
   DATA_EXCHANGE_APPROACH = 'dataExchangeApproach',
   MTO_INFO = 'modelToOperations',
-  MTO_CATEGORY = 'mtoCategory'
+  MTO_CATEGORY = 'mtoCategory',
+  MTO_MILESTONES = 'mtoMilestones'
```

Comment:
There's inconsistency with the plurality of this naming.  Either make both plural or both singular.